install_ubuntu.sh

```diff
@@ -16,13 +16,13 @@ rm -f ~/.joplin/*.AppImage ~/.local/share/applications/joplin.desktop
 # Creates the folder where the binary will be stored
 mkdir -p ~/.joplin/
 # Download the latest version
-wget -O ~/.joplin/Joplin-$version-x86_64.AppImage https://github.com/laurent22/joplin/releases/download/v$version/Joplin-$version-x86_64.AppImage 
+wget -O ~/.joplin/Joplin.AppImage https://github.com/laurent22/joplin/releases/download/v$version/Joplin-$version-x86_64.AppImage 
 # Gives execution privileges
-chmod +x ~/.joplin/Joplin-$version-x86_64.AppImage
+chmod +x ~/.joplin/Joplin.AppImage
 # Download icon
 wget -O ~/.joplin/Icon512.png https://joplin.cozic.net/images/Icon512.png
 # Create icon for Gnome
-echo -e "[Desktop Entry]\nEncoding=UTF-8\nName=Joplin\nExec=/home/$USER/.joplin/Joplin-$version-x86_64.AppImage\nIcon=/home/$USER/.joplin/Icon512.png\nType=Application\nCategories=Application;" >> ~/.local/share/applications/joplin.desktop
```

Comment:
The error is here:

You are using `>>` which appends to the current file. This is wrong. you want to create a new file (replacing the old).